pkgs/desktops/gnome/default.nix

```
@@ -73,8 +73,8 @@ lib.makeScope pkgs.newScope (self: with self; {
 
   gnome-settings-daemon = callPackage ./core/gnome-settings-daemon { };
 
-  # Using 3.38 to match Mutter used in Pantheon
-  gnome-settings-daemon338 = callPackage ./core/gnome-settings-daemon/3.38 { };
+  # Using 42 to match Mutter used in Pantheon
+  gnome-settings-daemon42 = callPackage ./core/gnome-settings-daemon/42 { };
```

```suggestion
  gnome-settings-daemon42 = callPackage ./core/gnome-settings-daemon/42.nix { };
```
The structure right now is a bit weird. Normally we would do this.

---

There are extra patches so we need to use a subdirectory.